CanonicalLtd/documentation-builder
# Add syntax highlighting to code

Add syntax highlighting to documents. Resolves https://github.com/ubuntudesign/docs.ubuntu.com/issues/28.

## QA

Clone down ubuntu-core-docs:

``` bash
git clone https://github.com/canonicalltd/ubuntu-core-docs
cd ubuntu-core-docs
```

Install this documentation builder branch:

``` bash
virtualenv env
git clone -b code-highlighting https://github.com/CanonicalLtd/documentation-builder/
env/bin/pip install documentation-builder
```

Run documentation builder:

``` bash
documentation-builder/bin/documentation-builder --force
```

Now open up the generated file at `build/en/guides/build-device/config-hooks.html` in a browser and see syntax highlighting in all its glory.

If you want, try adding more and different types of code to that file, using the GitHub code style:

    ``` <language-name>
    <some code>
    ```

And re-running documentation builder.
